goutubedl.go

```diff
@@ -330,6 +331,10 @@ func infoFromURL(
 		cmd.Args = append(cmd.Args, "--downloader", options.Downloader)
 	}
 
+	if options.Cookies != "" {
+		cmd.Args = append(cmd.Args, "--cookies", options.Cookies)
+	}
+
```

Comment:
Seem to work without having same code in `DownloadWithOptions`? maybe the cookie gets stored in the info json dump?

Reply:
yeah, works without having same code in `DownloadWithOptions`
I think cookie passed directly to cmd
```
2024/08/17 09:11:17 cmd   [yt-dlp_macos --ignore-errors --no-call-home --restrict-filenames --netrc --batch-file - --dump-single-json --cookies cookies.txt]
2024/08/17 09:11:53 cmd   [yt-dlp_macos --ignore-errors --no-call-home --newline --restrict-filenames --netrc --output - --load-info /var/folders/sometmpfolder/info.json -f best[ext=mp4]/best]
```

Reply:
Ok thanks for testing